go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -877,3 +895,29 @@ func (bsnc *BaseSecondaryNetworkController) requireDHCP(pod *corev1.Pod) bool {
 		bsnc.IsPrimaryNetwork() &&
 		bsnc.TopologyType() == types.Layer2Topology
 }
+
+func (bsnc *BaseSecondaryNetworkController) createVMSourcePodLSPOpsForLiveMigration(kubevirtLiveMigrationStatus *kubevirt.LiveMigrationStatus, nadName, switchName string) ([]ovsdb.Operation, error) {
+	if kubevirtLiveMigrationStatus.State != kubevirt.LiveMigrationTargetDomainReady {
+		return nil, nil
+	}
```

Comment:
not in this code path..
failed migration is only detected if the pod is Failed/Completed, meaning it will be at the `RevmoveResource` code path, not here. It is addressed at the last commit.

Reply:
Right, always forget about that.